comps/pulp-comps-rhel7.xml

```diff
@@ -52,6 +52,8 @@
       <packagereq type="default">python-pymongo</packagereq>
       <packagereq type="default">python-pymongo-gridfs</packagereq>
       <packagereq type="default">python-gridfs</packagereq>
+      <packagereq type="default">libsolv</packagereq>
+      <packagereq type="default">python-solv</packagereq>
```

Comment:
I think that's going to need to be python2-solv, since that's the name of the RPM in the scratch build:

python2-solv-0.6.34-3.el7.x86_64.rpm